contrib/craftscripts/clearblocks.js

```diff
@@ -0,0 +1,60 @@
+// Marks all blocks that have at least 3 exposed sides (adjacent to an air block)
+// and then deletes those blocks.
+// Based on jsa's MCEdit Filter clearBlocks.py
+
+importClass(Packages.com.sk89q.worldedit.world.block.BlockTypes)
+
+context.checkArgs(0, -1, "")
+
+var editSession = context.remember()
+var region = context.getSession().getRegionSelector(player.getWorld()).getRegion()
+
+var min = region.getMinimumPoint()
+var width = region.getWidth()
+var height = region.getHeight()
+var length = region.getLength()
+
+function sidesExposed(block) {
```

Comment:
This entire function can be replaced by the `#surface` mask, don't recall the exact implementing class but it should be sufficient. I would rather not add more clunky code for us to maintain in the craftscripts dir.